src/mantine-core/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -621,6 +634,8 @@ export const MultiSelect = forwardRef<HTMLInputElement, MultiSelectProps>(
                 value={searchValue}
                 onChange={handleInputChange}
                 onFocus={handleInputFocus}
+                onCompositionStart={handleCompositionStart}
```

Comment:
There is no need to create separate functions for this, please use inline functions instead to make component less bloated:
`onCompositionStart={() => setIMEOpen(true)}`

Reply:
Done